test/e2e/log.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
```

Comment:
do we want to store this file in git? or it's just a temporary testing script?

Reply:
this file and others was committed by accident